galaxy/templates/deployment-job.yaml

```diff
@@ -41,48 +41,14 @@ spec:
         {{- tpl (toYaml $.Values.jobHandlers.podSpecExtra) (set $ "handler_num" $handler_num) | nindent 6 }}
       {{- end }}
       initContainers:
-        - name: {{ $.Chart.Name }}-init-postgres
-          image: alpine:3.7
-          command: ['sh', '-c', 'chown 101:101 {{ $.Values.persistence.mountPath }}; until nc -z -w3 {{ template "galaxy-postgresql.fullname" $ }} 5432; do echo waiting for galaxy-postgres service; sleep 1; done;']
-          volumeMounts:
-            - name: galaxy-data
-              mountPath: {{ $.Values.persistence.mountPath }}
-        - name: {{ $.Chart.Name }}-init-mounts
-          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag | default $.Chart.AppVersion }}"
+        - name: {{ $.Chart.Name }}-wait-db
+          image: "{{ $.Values.image.repository }}:{{ $.Values.image.tag }}"
           imagePullPolicy: {{ $.Values.image.pullPolicy }}
-          command: ['sh', '-c', {{ include "galaxy.init-container-commands" $ | squote }}]
+          command: ['sh', '-c', '{{ include "galaxy.init-container-wait-command" $ }} sleep {{ $.Values.jobHandlers.startupDelay }}; sleep {{ $.Values.jobHandlers.startupDelay }};']
```

Comment:
Why 2 `sleep` commands?

Reply:
It shouldn't be oopsie, thanks for catching will fix now